.gitignore

```diff
@@ -6,6 +6,8 @@ __pycache__/
 # C extensions
 *.so
 
+city_weather.db
```

Comment:
The `city_weather.db` file is being ignored. Ensure that this is intentional and that this file does not need to be included in version control for collaboration or deployment purposes.